src/helpers/constants/validation.schemas.js

```diff
@@ -56,11 +56,21 @@ export default {
       .required(),
     profileImage: Joi.string(),
     facebook: Joi.string(),
-    twitter: Joi.string(),
+    twitter: Joi.string()
   }),
   rateArticle: Joi.object().keys({
     rating: number
       .valid(numbers.ONE, numbers.TWO, numbers.THREE, numbers.FOUR, numbers.FIVE)
       .required()
   }),
+  reportArticle: Joi.object().keys({
+    reason: number.required()
+  }),
+  updateArticle: Joi.object().keys({
+    title: Joi.string().allow(null, ''),
```

Comment:
@MCFrank16 we don't want articles with null titles or bodies

Reply:
okay lemme remove the null in there and allow it to only be empty